osx/release-docs/RELEASE_NOTES.md

```diff
@@ -1,6 +1,30 @@
 Release Notes for KA Lite Mac OS X Application
 ==============================================
 
+0.16.8
+------
+
+**Mac Application**
+
+* FIXED - Fix Unclean shutdown in OS X installer.
+
+0.16.7
+------
+
+**Mac Installer**
+
+* FIXED - Pyrun failed to install setuptools.
+
+0.16.1
+------
+
+**Mac Installer**
+
+* NEW - We now upgrade the KA Lite Monitor app.
```

Comment:
It would be good if you can expound the "We now upgrade the app." sentence here.  What were upgraded on the app?